jupyter-lib/lib-ext/src/main/kotlin/org/jetbrains/kotlinx/jupyter/ext/graph/GraphNode.kt

```diff
@@ -0,0 +1,10 @@
+package org.jetbrains.kotlinx.jupyter.ext.graph
+
+interface GraphNode<T> {
+    val node: T
```

Comment:
Does graphNode contain the "node"? Do you mean additional information related to the Node? Unclear naming